sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -173,6 +174,14 @@ export class StudentApplicationOfferingChangeRequestAPIInDTO {
   applicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus;
 }
 
+export class ApplicationOfferingChangeAssessmentAPIInDTO {
+  @IsNotEmpty()
+  @MaxLength(NOTE_MAX_LENGTH)
+  note: string;
+  @IsEnum(ApplicationOfferingChangeRequestStatus)
```

Comment:
I missed to mention in the student part of the story. `@IsIn` will be more accurate here.

```ts
@IsIn([ApplicationOfferingChangeRequestStatus.Approved, ApplicationOfferingChangeRequestStatus.DeclinedBySABC]
```